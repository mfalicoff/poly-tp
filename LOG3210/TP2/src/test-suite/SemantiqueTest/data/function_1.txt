Declaration

EndDeclaration

bool a(){
    return 1 + 2;
}