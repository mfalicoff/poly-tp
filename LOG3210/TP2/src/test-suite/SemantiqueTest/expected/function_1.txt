Return type does not match function type